hack/use-openshift-prometheus.sh

```diff
@@ -216,6 +216,12 @@ EOM
   fi
 
   echo "Apply Telemetry resource to [${ISTIO_NAMESPACE}] to ensure Prometheus is a metrics provider"
+  tel_name=$(${OC} get telemetry -n ${ISTIO_NAMESPACE} -o name)
+  if [ ! -z "$tel_name" ]
+  then
+    echo "Found existing ${tel_name} in ${ISTIO_NAMESPACE}, patching..."
+    ${OC} patch ${tel_name} -n ${ISTIO_NAMESPACE} --type=merge -p '{"spec":{"metrics":[{"providers":[{"name":"prometheus"}]}]}}'
+  else
   cat <<EOM | ${OC} apply -n ${ISTIO_NAMESPACE} -f -
```

Comment:
This is to be part of the `else` clause, correct? Needs to be indented.

```suggestion
    cat <<EOM | ${OC} apply -n ${ISTIO_NAMESPACE} -f -
```